src/context/Confirmation/index.tsx

```diff
@@ -48,6 +33,8 @@ const ConfirmationModalContextProvider = ({ children }: BaseComponentProps) => {
     const [showConfirmationModal, setShowConfirmationModal] = useState(false);
     const resolver = useRef<any>();
 
+    const [continueAllowed, setContinueAllowed] = useState<boolean>(true);
```

Comment:
need to be able to disable clicking continue until the user selects a materialization. So adding this here and then allowing the control of it with the context as well. not my favorite solution due to extra renders... but not the worst.